src/contained-list/contained-list.stories.ts

```diff
@@ -0,0 +1,333 @@
+				<ng-container ibmContainedListItemButton>List item</ng-container>
+			</ibm-contained-list-item>
+			<ibm-contained-list-item [clickable]="true" [action]="action">
+				<ng-container ibmContainedListItemButton>List item</ng-container>
+			</ibm-contained-list-item>
+		</ibm-contained-list>
+	`
+});
+export const withInteractiveItemsAndActions =
+	withInteractiveItemsAndActionsTemplate.bind({});
+
+const withLayerTemplate: Story<ContainedList> = () => ({
+	template: `
+		<div ibmStack="vertical" [gap]="5">
+			<ibm-contained-list label="List title">
+				<ibm-contained-list-item>List item</ibm-contained-list-item>
+				<ibm-contained-list-item>List item</ibm-contained-list-item>
+				<ibm-contained-list-item>List item</ibm-contained-list-item>
+				<ibm-contained-list-item>List item</ibm-contained-list-item>
+			</ibm-contained-list>
+			<div [ibmLayer]="1">
+				<div ibmStack="vertical" [gap]="5">
+					<ibm-contained-list label="List title">
+						<ibm-contained-list-item>List item</ibm-contained-list-item>
+						<ibm-contained-list-item>List item</ibm-contained-list-item>
+					</ibm-contained-list>
+					<div [ibmLayer]="2">
+						<ibm-contained-list label="List title">
+							<ibm-contained-list-item>List item</ibm-contained-list-item>
+							<ibm-contained-list-item>List item</ibm-contained-list-item>
+						</ibm-contained-list>
+					</div>
+				</div>
+			</div>
+		</div>
+	`
+});
+export const withLayer = withLayerTemplate.bind({});
+
+const withListTitleDecoratorsTemplate: Story<ContainedList> = () => ({
+	template: `
+		<ng-template #label let-item>
+			<div style="display: flex; alignItems: center; justifyContent: space-between">
+		    	<span>List title</span>
+		    	<ibm-tag size="sm">4</ibm-tag>
+		    </div>
+		</ng-template>
+		<ibm-contained-list [label]="label">
+			<ibm-contained-list-item>List item</ibm-contained-list-item>
+			<ibm-contained-list-item>List item</ibm-contained-list-item>
+			<ibm-contained-list-item>List item</ibm-contained-list-item>
+			<ibm-contained-list-item>List item</ibm-contained-list-item>
+		</ibm-contained-list>
+	`
+});
+export const withListTitleDecorators = withListTitleDecoratorsTemplate.bind({});
+
+const playgroundTemplate: Story<ContainedList> = (args) => ({
+	props: args,
+	template: `
+		<ibm-contained-list [label]="label" [kind]="kind" [size]="size" [isInset]="isInset">
+			<ibm-contained-list-item>List item</ibm-contained-list-item>
+			<ibm-contained-list-item>List item</ibm-contained-list-item>
+			<ibm-contained-list-item>List item</ibm-contained-list-item>
+			<ibm-contained-list-item>List item</ibm-contained-list-item>
+			<ibm-contained-list-item>List item</ibm-contained-list-item>
+			<ibm-contained-list-item>List item</ibm-contained-list-item>
+			<ibm-contained-list-item>List item</ibm-contained-list-item>
+			<ibm-contained-list-item>List item</ibm-contained-list-item>
+		</ibm-contained-list>
+	`
+});
+export const playground = playgroundTemplate.bind({});
```

Comment:
I don't think we need this, React refers to stories with knobs as `playground`. We automatically add knobs using compodoc.


```suggestion
```